### ISSUE TITLE
Unify vocabulary list views

### ISSUE THREAD

**Issue by IvyJL** [Hedera-Lang-Learn/hedera#498]
I would like to have the same options in other vocabulary lists that I have with the Personal Vocab List: 

1. sorting
2. searching
3. associating with a specific lemma
4. exporting

**Comment by jaguillette** [comment]
The vocab lists should be unified on dev, albeit with the bug on the vocab list quick add that we know about (#502). Once we get that bug fixed that unification will be part of what's deployed to production.

**Comment by IvyJL** [comment]
Yes, the vocab list views look unified on dev and I think the quick add bug is fixed now too.
